roles/zuul/files/jobs/lore.yaml

```diff
@@ -6,6 +6,10 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
+          sudo apt-get install -yqq curl
+          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
```

Comment:
You don't think our `ubuntu-xenial` image should have node? I'm not worried about performance tuning our builds yet, but I do think it would be a good idea to have some basic tools available.

I'm fine with this for now, but we will need to have images preloaded with software for projects to install their dependencies at some point.